apps/passport-server/src/database/queries/telegram/insertTelegramConversation.ts

```diff
@@ -8,14 +8,15 @@ export async function insertTelegramVerification(
   client: Pool,
   telegramUserId: number,
   telegramChatId: number,
-  semaphoreId: string
+  semaphoreId: string,
+  telegramUsername?: string
```

Comment:
note that `telegramUsername` can be undefined, because some users might not have usernames. So there's always a chance the value will be null